package.json

```diff
@@ -4,17 +4,19 @@
   "description": "Twilio Video Room Monitor Javascript Library",
   "main": "dist/node/index.js",
   "types": "dist/node/index.d.ts",
+  "regular": "dist/browser/twilio-video-room-monitor.js",
+  "minified": "dist/browser/twilio-video-room-monitor.min.js",
   "scripts": {
     "start": "tsc --watch",
     "test": "cross-env TZ=utc jest",
     "test:ci": "cross-env TZ=utc jest --ci --runInBand --reporters=default --reporters=jest-junit --coverage --silent",
     "lint": "eslint src",
-    "parcel:watch": "cross-env PARCEL_TARGET=browser parcel src/index.tsx --no-hmr --out-dir dist/browser --out-file twilio-video-room-monitor.js",
+    "parcel:watch": "cross-env PARCEL_TARGET=browser parcel src/index.tsx --no-hmr --dist-dir dist/browser --target regular",
     "prebuild": "rimraf dist/",
     "build": "npm run build:node && npm run build:browser && npm run build:browser:min",
     "build:node": "tsc",
-    "build:browser": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --out-dir dist/browser --out-file twilio-video-room-monitor.js --target browser --no-minify",
-    "build:browser:min": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --out-dir dist/browser --out-file twilio-video-room-monitor.min.js --target browser",
+    "build:browser": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --no-scope-hoist --target regular",
+    "build:browser:min": "cross-env PARCEL_TARGET=browser parcel build src/index.tsx --no-source-maps --no-scope-hoist --target minified",
```

Comment:
I'm not having any issues without `--no-scope-hoist`. Not sure why. Mind taking another look at this?

Reply:
removed it and it works for me now too